### PR TITLE
Wrote an intro tutorial for beginners.

### LINKED ISSUE #314: Write better 'getting started' guide
I'm getting feedback that our docs are a bit hard for newcomers to grok. I'd like to improve this.

* Start a very quick summary & some links to reducer basics.
* Then show hooking up the store
* Then show an `@select`
* Then show something with `store.select`
* Then show routing
* Then show forms
* Then show an epic.

### PR DESCRIPTION
**Pull request by SethDavenport**
Fixes #314

### CODE REVIEW

**Comment by SethDavenport** [comment]
@rajinder-yadav this was somthing you mentioned a while back. Thoughts?

**Comment by SethDavenport** [comment]
Note to self: 
* fix some typos
* redux-beacon is another cool middleware example
* is it worth talking about combineReducers?
* if we put dispatch in the action service then the component is much cleaner

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Angular components. In your top-level app module, you
+can configure your Redux store with reducers, initial state,
+and optionally middlewares and enhancers as you would in Redux directly.
+
+```typescript
+import { NgReduxModule, NgRedux } from '@angular-redux/store';
+import reduxLogger from 'redux-logger';
+import { rootReducer } from './reducers';
+
+interface IAppState { /* ... */ };
```
Comment (rajinder-yadav) [inline docs/quickstart.md]: Would be nice to expand on how the interface `IAppState` comes into play. For single state or multiple states. Later how those states are accessed, if they are separate groups of states.

File: docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+@NgModule({
+  /* ... */
+  imports: [ /* ... */, NgReduxModule ]
+})
+export class AppModule {
+  constructor(ngRedux: NgRedux<IAppState>) {
+    ngRedux.configureStore(rootReducer, {}, [ createLogger() ]);
```
Comment (rajinder-yadav) [inline docs/quickstart.md]: I found `configureStore` initially confusing, the API and what each argument signifies. Would be nice to expand and explain it here quickly.

**Comment by rajinder-yadav** [comment]
Looks goods, just a few comments.

File: README.md [hunk 3]
```diff
@@ -8,116 +8,32 @@ For Angular 1 see [ng-redux](https://github.com/wbuchwalter/ng-redux)
...
+
+We provide a set of npm packages that help you integrate your redux store
+into your Angular 2+ applications. Our approach helps you by bridging the gap
+with some of Angular's advanced features, including:
+
+* Change processing with RxJS observables
+* Compile time optimizations with `NgModule` and Ahead-of-Time compilation
+* Integration with the Angular change detector.
+
+## Getting Started
```
Comment (SethDavenport) [inline README.md]: Yeah I will stick with English Title Case rules. Thanks.

Comment (SethDavenport) [inline README.md]: Although strictly speaking title case is typically defined in English as capitalizing everything except prepositions, articles, and pronouns :)

File: docs/quickstart.md (hunk 1 above)
Comment (SethDavenport) [inline docs/quickstart.md]: Yeah I'll add a note. However I didn't want to get into too complicated an example in this tutorial. I might write a second chapter later.

**Pull request merged**